### PR TITLE
COP21 OPUs, message refactor, linting

### PR DESCRIPTION
**Pull request by jacksonsj**
**IMPORTANT: Please do not create a Pull Request without creating an issue first.**

## Affected Process Elements
- [ ] Data Pack generation
- [ ] Data Pack model
- [ ] Data Pack self-service app
- [ ] Data Pack processing
- [ ] Site Tool model/distribution
- [ ] Site Tool generation (from Data Pack)
- [ ] Site Tool generation (from DATIM - for OPUs)
- [ ] Site Tool self-service app
- [ ] Site Tool to Data Pack comparison self-service app
- [ ] Site Tool to DATIM comparison self-service app
- [ ] Site Tool processing
- [ ] Site Tool import

## Types of changes

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe): 

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

## Related Issues

- Resolves #01
- Fixes #02
- Closes #03

## Final Checklist

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by jacksonsj** [comment]
@jason-p-pickering @sam-bao - All outstanding code changes should now be in `dev`. Could you review for pushing into `master`?

File: DESCRIPTION [hunk 1]
```diff
@@ -52,14 +52,17 @@ Imports:
...
+    fuzzyjoin (>= 0.1.6),
+    R6
 Suggests:
     devtools (>= 2.0.1),
     httptest,
-    testthat
+    testthat,
+    cellranger,
+    rprojroot
 Remotes: jason-p-pickering/datim-validation,
```
Comment (jacksonsj) [inline DESCRIPTION:63]: @jason-p-pickering - I've seen some places where we point to `pepfar-datim/datim-validation`, and others to `jason-p-pickering/datim-validation`. Which is best?

Comment (jason-p-pickering) [reply DESCRIPTION]: I have fixed this now, as I was causing a merge conflict with master. It should be pepfar-datim/datim-validation.

**Review by jason-p-pickering — APPROVED** [review]
With a PR of this size, it is obviously quite tough to make comments on everything! But, I believe we are in good shape now, having testing with different Datapacks and OPUs with the app. I suggest we go ahead and merge this and work out any bugs at a later stage.

**Pull request merged**